Backend/app/rider-platform/rider-app/Main/src/Domain/Types/Booking/Type.hs

```diff
@@ -33,8 +33,11 @@ import Kernel.Types.Id
 import Servant.API
 import Tools.Beam.UtilsTH
 
+activeBookingStatusForCheck :: [BookingStatus]
+activeBookingStatusForCheck = [NEW, CONFIRMED, AWAITING_REASSIGNMENT, TRIP_ASSIGNED]
+
 activeBookingStatus :: [BookingStatus]
-activeBookingStatus = [NEW, CONFIRMED, AWAITING_REASSIGNMENT, TRIP_ASSIGNED]
+activeBookingStatus = [AWAITING_REASSIGNMENT, TRIP_ASSIGNED]
```

Comment:
Where is this being used? I don't see any difference in prev and current code in terms of functionality. What is this solving?

Reply:
Since the variable name is same as before so its changes can't be seen here